src/fetchers/stats-fetcher.js

```diff
@@ -123,7 +126,10 @@ async function fetchStats(
     rank: { level: "C", score: 0 },
   };
 
-  let res = await retryer(fetcher, { login: username });
+  let res = await retryer(fetcher, { 
+    login: username, 
+    starttime: year ? `${year}-01-01T00:00:00Z` : undefined
```

Comment:
This does not work without updating the [fetcher](https://github.com/anuraghazra/github-readme-stats/blob/166972d7e4f12ab17ee7aa2b2000d784444bd9fd/src/fetchers/stats-fetcher.js#L24-L63). See https://github.com/anuraghazra/github-readme-stats/pull/1691/files for an example on how to fetch contributions per year.

https://github.com/anuraghazra/github-readme-stats/blob/ee8ce91b7f04a4876908e9cbce4a3b022ef475c7/src/fetchers/stats-fetcher.js#L94-L108